common-test/src/main/java/io/strimzi/test/k8s/KubeClient.java

```diff
@@ -89,12 +89,20 @@ default K delete(String... files) {
     ProcessResult exec(String pod, String... command);
 
     /**
-     * Wait for the deployment with the given {@code name} to
+     * Execute the given {@code command} in the kubernetes workspace.
+     * @param command The command
+     * @return The process result.
+     */
+    ProcessResult execInKubeWorkspace(String... command);
```

Comment:
In kubernetes it's called a "namespace" (an OpenShift workspace is a namespace + some custom annotations, I believe). But I'm confused by the `...InKubeWorkspace` part. The impl just calls `exec`, so this is just running an executable outside the k8s/os cluster, right? So in what sense is that in any workspace?

Reply:
Agree that we can call this method just `exec`. TBH I just wanted to say by this name that we will NOT execute a command on some Kubernetes pod. But name `exec` has more sense